### ISSUE TITLE
gnt-cluster rename fails

### ISSUE THREAD

**Issue by Ganeti-Issues-Migrator** [ganeti/ganeti#83]
Originally reported of Google Code with ID 27.

```
gnt-cluster rename serviceold.cassiopeaonline.net
This will rename the cluster to 'serviceold.cassiopeaonline.net'. If
you are connected over the network to the cluster name, the operation
is very dangerous as the IP address will be removed from the node and
the change may not go through. Continue?
y/[n]/?: y
Traceback (most recent call last):
  File "/usr/sbin/gnt-cluster", line 361, in ?
    sys.exit(GenericMain(commands, override={"tag_type":
constants.TAG_CLUSTER}))
  File "/var/lib/python-support/python2.4/ganeti/cli.py", line 493, in
GenericMain
    result = func(options, args)
  File "/usr/sbin/gnt-cluster", line 87, in RenameCluster
    SubmitOpCode(op)
  File "/var/lib/python-support/python2.4/ganeti/cli.py", line 386, in
SubmitOpCode
    return proc.ExecOpCode(op)
  File "/var/lib/python-support/python2.4/ganeti/mcpu.py", line 128, in
ExecOpCode
    hm = HooksMaster(rpc.call_hooks_runner, self, lu)
  File "/var/lib/python-support/python2.4/ganeti/mcpu.py", line 213, in
__init__
    self.env, node_list_pre, node_list_post = self._BuildEnv()
  File "/var/lib/python-support/python2.4/ganeti/mcpu.py", line 233, in
_BuildEnv
    lu_env, lu_nodes_pre, lu_nodes_post = self.lu.BuildHooksEnv()
  File "/var/lib/python-support/python2.4/ganeti/cmdlib.py", line 974, in
BuildHooksEnv
    env = {
AttributeError: 'OpRenameCluster' object has no attribute 'sstore'

```
Originally added on 2008-04-11 13:35:56 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
What version is this against? If 1.2, this is indeed a bug, but was fixed and will be
released in 1.2.4
```
Originally added on 2008-04-11 14:51:19 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
That happened with in 1.2.3. 
```
Originally added on 2008-04-13 10:27:09 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
Ok then, we'll leave this open until 1.2.4 is released.
```
Originally added on 2008-04-14 10:05:17 +0000 UTC.

Changed State: Started

**Comment by Ganeti-Issues-Migrator** [comment]
```
This was fixed in 1.2.4
```
Originally added on 2008-07-01 19:55:05 +0000 UTC.

Changed State: Fixed

**Comment by Ganeti-Issues-Migrator** [comment]
```
-- Empty comment --
```
Originally added on 2012-10-22 08:57:07 +0000 UTC.

Changed State: Released
